client/src/components/layouts/global.css

```diff
@@ -4,6 +4,25 @@ html {
   -webkit-font-smoothing: antialiased;
 }
 
+[dir='rtl'] .nav-list {
+  right: 0;
+}
+
+.fcc_searchBar .ais-SearchBox-input,
+.fcc_searchBar .ais-Hits {
+  padding-inline-start: 1.5em;
+}
+@media (max-width: 980px) {
+  [dir='rtl'] #universal-nav-logo {
+    left: inherit;
+    right: 17px;
+  }
+  [dir='rtl'] .ais-SearchBox-submit {
+    left: inherit;
+    right: 0.55em;
+  }
+}
+
```

Comment:
because our new system of approving staying, I can add this values and people can test it without hassle. 

And when it gets two approvals, we can delete them and merge it.

These values are RTL related values, because this PR isn't about RTL, but making the search bar easier to mirror, this shouldn't be merged. 

But I don't want to make reviewing harder so I kept them for reviewing propose

```suggestion
```

Reply:
I don't know why I have put the padding value was on here and not in `search.css`.

In RTL styling file, I should add to account for new position `absolute` in the icon


```css
@media (min-width: 981px) {
 [dir="rtl"] .ais-SearchBox-submit {
    right: 0;
  }
}
```